fbx_document.cpp

```diff
@@ -1791,31 +1818,49 @@ Error FBXDocument::_parse_meshes(Ref<FBXState> p_state) {
 								array_copy[l] = array[l];
 							}
 
-							if (fbx_shape->position_offsets.count > 0) {
-								Vector<Vector3> varr;
-								const Vector<Vector3> src_varr = array[Mesh::ARRAY_VERTEX];
-								const int size = src_varr.size();
-								ERR_FAIL_COND_V(size == 0, ERR_PARSE_ERROR);
-								{
-									varr.resize(size);
-
-									Vector3 *w_varr = varr.ptrw();
-									const Vector3 *r_varr = varr.ptr();
-									for (int l = 0; l < size; l++) {
-										int32_t vertex_index = fbx_mesh->vertex_indices[uint32_t(indices[l])];
-										Vector3 blender_shape_offset = _as_vec3(ufbx_get_blend_shape_vertex_offset(fbx_shape, vertex_index));
-										w_varr[l] = r_varr[l] + blender_shape_offset;
+							Vector<Vector3> varr;
+							Vector<Vector3> narr;
+							const Vector<Vector3> src_varr = array[Mesh::ARRAY_VERTEX];
+							const Vector<Vector3> src_narr = array[Mesh::ARRAY_NORMAL];
+							const int size = src_varr.size();
+							ERR_FAIL_COND_V(size == 0, ERR_PARSE_ERROR);
+							{
+								varr.resize(size);
+								narr.resize(size);
+
+								Vector3 *w_varr = varr.ptrw();
+								Vector3 *w_narr = narr.ptrw();
+								const Vector3 *r_varr = src_varr.ptr();
+								const Vector3 *r_narr = src_narr.ptr();
```

Comment:
```suggestion
```
This variable is unused?

Reply:
Whoops, I read straight from `src_narr` instead. I think this is just an optimization in the original source that I kept.